code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -137,8 +137,10 @@
 
 /obj/item/weapon/storage/briefcase/false_bottomed/smg/New()
 	..()
-	var/obj/item/weapon/gun/projectile/automatic/SMG = new
+	var/obj/item/weapon/gun/projectile/automatic/uzi/SMG = new
 	SMG.gun_flags &= ~AUTOMAGDROP //dont want to drop mags in null space, do we?
+	var/obj/item/ammo_storage/magazine/uzi45/extended/EXT = new
+	SMG.stored_magazine = EXT
```

Comment:
delete the old stored magazine first so it can get garbage collected

Reply:
there is no old magazine stored inside, it gets replaced because stored magazine isn't a list(), I've tested it

Reply:
in the parent new() it creates a magazine of type `/obj/item/ammo_storage/magazine/uzi45`

so it's creating an old magazine then immediately overwriting it... but that magazine is now in nullspace with nothing pointing to it